docs/CHANGELOG.rst

```diff
@@ -9,6 +9,10 @@ All notable changes to this project are documented in this file.
 Unreleased
 ==========
 
+Added
+-----
+ - Add docker-compose configuration for PostgreSQL
+
 Changed
```

Comment:
Please, also mention that the default database user and port in test project's settings have changed.